### PR TITLE
Mk/better nested print

### PR DESCRIPTION
**Pull request by 7sne**

### CODE REVIEW

File: src/components/DecodedCalldataTree.tsx [hunk 1]
```diff
@@ -57,28 +57,58 @@ function CalldataTreeNode({
     return (
       <span className={className}>
         <code>
-          {node.name ? (
-            <b className="text-deth-pink"> {node.name} </b>
+          {node.type.match(/[[*\]]/) ? (
```
Comment (LogvinovLeon) [inline src/components/DecodedCalldataTree.tsx:60]: Can we please have a convention that we name all the regexes that are not trivial?
I think this matches: `[`, `]`, `*` but it takes some time to mentally parse and I still don't get why there is a star in a char set.
Let's have smth like: `arrayOrStructRegex` or whatever this actually is.

**Review by LogvinovLeon — CHANGES_REQUESTED** [review]
Otherwise LGTM and works like a bless

File: src/components/DecodedCalldataTree.tsx (hunk 1 above)
Comment (hasparus) [reply src/components/DecodedCalldataTree.tsx:60]: I have a stronger ask.
Could we comment all non trivial regexes with examples? :D

Comment (7sne) [reply src/components/DecodedCalldataTree.tsx:60]: fine I will extract it from here 😆

**Review by hasparus — APPROVED** [review]

**Review by LogvinovLeon — APPROVED** [review]

**Pull request merged**